### ISSUE TITLE
How to plot train vs. val loss over epochs

### ISSUE THREAD

**Issue by fabioziegler** [krasserm/super-resolution#96]
Hello,

how can I plot train vs. val loss over epochs? The trainer.train call does not return a history object. I'm using WDSR B model.

Thank you.
